objectivec/GPBWellKnownTypes.h

```diff
@@ -96,4 +115,82 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+#pragma mark - GPBAny
+
+/**
+ * Category for GPBAny to help work with the message within the object.
+ **/
+@interface GPBAny (GBPWellKnownTypes)
+
+/**
+ * Returns a new instance of GPBAny that includes the serialized message. Uses
+ * the type.googleapis.com/ type_url prefix. On failure returns nil and fills
+ * in the optional errorPtr with the details. The error can be a
+ * GPBWellKnownTypesErrorDomain error or a general message serialization error.
+ */
++ (nullable instancetype)anyWithMessage:(nonnull GPBMessage *)message
+                                  error:(NSError **)errorPtr;
+
+/**
+ * Returns a new instance of GPBAny that includes the serialized message. On
+ * failure returns nil and fills in the optional errorPtr with the details. The
+ * error can be a GPBWellKnownTypesErrorDomain error or a general message
+ * serialization error.
+ */
++ (nullable instancetype)anyWithMessage:(nonnull GPBMessage *)message
+                          typeURLPrefix:(nonnull NSString *)typeURLPrefix
+                                  error:(NSError **)errorPtr;
+
+/**
+ * Serializes message and initializes the instance to contain the info. Uses
+ * the type.googleapis.com/ type_url prefix.  On failure returns nil and fills
+ * in the optional errorPtr with the details. The error can be a
+ * GPBWellKnownTypesErrorDomain error or a general message serialization error.
+ */
+- (nullable instancetype)initWithMessage:(nonnull GPBMessage *)message
```

Comment:
/**
Initializes a GPBAny containing the serialized message. This uses
type.googleapis.com/ as the type_url's prefix.

@ param message The message to be packed into the GPBAny.
@ param errorPtr   Pointer to an error that will be populated if something goes wrong.

@ return A newly configured GPBAny with the given message, or nil on failure.
*/